site/static/docs/5.3/assets/img/boosted-sprite.svg

```diff
@@ -1,56 +1,61 @@
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 24 24" id="info">
-    <path d="M12 1.812c-5.633 0-10.2 4.567-10.2 10.2 0 5.633 4.567 10.2 10.2 10.2 5.633 0 10.2-4.567 10.2-10.2 0-5.633-4.567-10.2-10.2-10.2Zm0 2.4a1.8 1.8 0 1 1 0 3.6 1.8 1.8 0 0 1 0-3.6Zm3 14.4H9.6v-.618l.127-.004c.373-.012.65-.118.825-.315.062-.071.167-.314.167-1.132v-5.452c0-.778-.109-1.07-.2-1.18-.128-.153-.396-.243-.796-.268L9.6 9.636v-.624h4.2v7.531c0 .778.203 1.071.294 1.18.128.153.376.243.783.267l.123.008v.614Z"/>
+  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="info">
+    <path d="M500 75.5c-234.72 0-425 190.28-425 425s190.28 425 425 425 425-190.28 425-425-190.28-425-425-425m0 100a75 75 0 1 1-75 75 75 75 0 0 1 75-75m125 600H400v-25.737l5.287-.167c15.555-.5 27.115-4.919 34.368-13.136 2.576-2.972 6.956-13.1 6.956-47.153v-227.2c0-32.392-4.523-44.6-8.311-49.141-5.359-6.391-16.516-10.147-33.17-11.156l-5.13-.31v-26h175v313.807c0 32.415 8.457 44.618 12.24 49.142 5.34 6.389 15.68 10.142 32.625 11.152l5.135.3v25.6Z" style="fill-rule:evenodd"/>
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 24 24" id="warning">
-    <path d='M19.875 21H4.1a2.25 2.25 0 0 1-1.946-3.384l.019-.031v-.002l7.868-13.467a2.255 2.255 0 0 1 3.892 0l7.887 13.5A2.249 2.249 0 0 1 19.875 21Z'/><path fill="#000" d='M12.012 18.6a1.14 1.14 0 1 1 0-2.28 1.14 1.14 0 0 1 0 2.28Zm1.137-10.177-.377 6.195c0 .42-.34.577-.76.577s-.76-.157-.76-.577l-.378-6.21a1.14 1.14 0 1 1 2.275.014Z'/>
+  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="warning-important-accessible">
+    <path d="M499.352 125a93.95 93.95 0 0 1 81.072 46.494L908.91 733.919A93.676 93.676 0 0 1 827.838 874.9H170.867A93.711 93.711 0 0 1 89.8 733.919l.8-1.3v-.1l327.681-561.025A93.83 93.83 0 0 1 499.352 125"/>
+    <path fill="#000" d="M500.352 679.926a47.494 47.494 0 1 0 47.484 47.494 47.53 47.53 0 0 0-47.484-47.494m0-379.949a47.53 47.53 0 0 0-47.483 47.493 28 28 0 0 0 .1 2.9l15.694 258.665v.5c.3 17.1 14.395 23.5 31.689 23.5 17.494 0 31.689-6.6 31.689-24l15.695-258.065.1-.9a23 23 0 0 0 .1-2.6 47.68 47.68 0 0 0-47.583-47.493"/>
   </symbol>
-  <symbol fill="currentColor" viewBox="0 0 24 24" id="danger">
-    <path d="m21.832 17.617-7.886-13.5a2.256 2.256 0 0 0-3.892 0L2.186 17.583v.002l-.02.032h.001A2.248 2.248 0 0 0 4.113 21h15.773a2.25 2.25 0 0 0 1.946-3.383Zm-6.88-.714a.823.823 0 0 1-1.163 0l-1.93-1.932-1.93 1.932a.823.823 0 0 1-1.174-1.155l.01-.01 1.93-1.93-1.861-1.863a.823.823 0 0 1 1.163-1.164l1.862 1.862 1.861-1.862a.823.823 0 0 1 1.174 1.153l-.01.011-1.862 1.862 1.93 1.931a.824.824 0 0 1 0 1.165Z"/>
+  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="error-severe">
```

Comment:
I can see a lot of modifications here to have `viewBox="0 0 1000 1000"` versions. Don't we need to update the warning, success, error, etc. icons in `_variables.scss`? (e.g. `$warning-icon-filled` etc.)

Reply:
Discussed together: not to be modified for now because the risk is too big to make a mistake and the difference is tiny... 
The differences between Boosted and designs (V6 kit) should be tracked in an issue though (in particular paddings that do not include borders)

Reply:
Yep, let's create a dedicated issue after having merged this PR to tackle it for the next release